modelscope/pipelines/builder.py

```diff
@@ -128,7 +128,11 @@ def pipeline(task: str = None,
             pipeline_props = {'type': pipeline_name}
 
     pipeline_props['model'] = model
-    pipeline_props['device'] = device
+    # 'device' not in 'AutomaticSpeechRecognitionPipeline.get_cmd()', but ngpu
```

Comment:
this logic shold be added in the __init__ function of AutomaticSpeechRecognitionPipeline,  other pipeline use the device gpu/cpu as expected values